### DISCUSSION TITLE
Property '$queryRaw' does not exist on type 'unique symbol'.

### DISCUSSION THREAD

**Discussion by rizqonmaulana** [prisma/prisma#16667] [Q&A] [answered]
Hi, I am new to prisma. I am using prisma with nest.js & MySQL database.

I want to create a raw query but I don't know why the terminal always told me that: Property '$queryRaw' does not exist on type 'unique symbol'.

I think it's just a small issue but I have searched on the internet and still have no idea about this

prisma version 4.7.1

<img width="973" alt="image" src="https://user-images.githubusercontent.com/57583229/206067189-ca6bfc63-34cd-4205-b587-4462b6d54cbf.png">

**Comment by nurul3101** [comment, marked as answer]
Hello @rizqonmaulana 👋 
You are getting this error because you are not initiating the `PrismaClient`.

You can instantiate PrismaClient like this
```typescript
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

export async function encrypt(data: string) {
  const result =
    await prisma.$queryRaw`SELECT * FROM Accounts WHERE uid = 3225`;

  return result;
}
```

Here's a [Quickstart](https://www.prisma.io/docs/getting-started/quickstart#4-explore-how-to-send-queries-to-your-database-with-prisma-client) which should be helpful to you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
